src/main/java/com/itschool/eventmanagment/repositories/EventRepository.java

```diff
@@ -4,6 +4,12 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.time.LocalDateTime;
+import java.util.List;
+
 @Repository
 public interface EventRepository extends JpaRepository<Event, Long > {
+    List<Event> findByDateTimeBetweenOrLocationContainingIgnoreCaseOrderByDateTime(LocalDateTime fromDateTime, LocalDateTime toDateTime, String location);
+    List<Event> findByDateTimeBetweenOrLocationContainingIgnoreCaseOrderByLocationAsc(LocalDateTime fromDateTime, LocalDateTime toDateTime, String location);
```

Comment:
Logic:
- We are using here `OR` operator, which means it will return results that either match the date range `OR` the location. Is this the behavior we want to achieve here? If we want to find events that match both criteria, we should use `AND` operator instead
```
List<Event> findByDateTimeBetweenAndLocationContainingIgnoreCase(LocalDateTime from, LocalDateTime to, String location);
```

Parameters name:
- I've noticed an inconsistency on the parameter's name; there is a particular reason why in other classes we have `for, to, location` and here use `forDateTime, toDateTime,...`? 
We already have the type of the parameter decided. Going on the same logical pattern, why we shouldn't have `locationString` either? Maybe we can simply the naming and we can be compliant with the rest of the code.